docker_client.go

```diff
@@ -97,7 +97,7 @@ func splitDockerImage(img string) (string, string, string) {
 func getContainers(client *docker.Client) ([]*RuntimeContainer, error) {
 
 	apiContainers, err := client.ListContainers(docker.ListContainersOptions{
-		All:  false,
+		All:  true,
```

Comment:
That seems like a fine solution to me.

I'd consider introducing a `Generator` struct that contains the `docker.Client` and the `Config` and moving the `generate*` functions to hang off of that.

@jwilder What do you think?